src/DynamoCore/Models/ModelBase.cs

```diff
@@ -201,15 +202,14 @@ public bool HandleModelEvent(string eventName, UndoRedoRecorder recorder)
         /// method takes only string input (the way they appear in DynamoTextBox),
         /// which overridden method can use for value conversion.
         /// </summary>
-        /// <param name="name">The name of a property/value to update.</param>
-        /// <param name="value">The new value to be set. This value comes directly
+        /// <param name="updateValueParams">It comprises 1. the name of a property/value to update.
+        /// 2. The new value to be set. This value comes directly
         /// from DynamoTextBox after user commits it. Overridden methods then use 
         /// a specific IValueConverter to turn this string into another data type 
-        /// that it expects.</param>
-        /// <param name="recorder"></param>
+        /// that it expects, 3. element resolver</param>
```

Comment:
Thanks for updating these comments, @aparajit-pratap! I'd request this to be further updated to:

```csharp
/// <param name="updateValueParams">Please see UpdateValueParams for details.</param>
```